ddtrace/_trace/tracer.py

```diff
@@ -283,9 +282,7 @@ def __init__(
             register_on_exit_signal(self._atexit)
 
         self._hooks = _hooks.Hooks()
-        atexit.register(self._atexit)
         forksafe.register_before_fork(self._sample_before_fork)
-        forksafe.register(self._child_after_fork)
```

Comment:
Might be going crazy here but do we replace this forksafe registering the tracer `child_after_fork` anywhere?

Reply:
The product plugin interface has that logic encapsulated so we don't need to repeat it in all products

Reply:
The product object has
```
def restart(join=False):
    from ddtrace.trace import tracer

    if tracer.enabled:
        tracer._child_after_fork()
```